contracts/position/IncreasePositionUtils.sol

```diff
@@ -99,7 +99,7 @@ library IncreasePositionUtils {
             );
         }
 
-        (cache.priceImpactUsd, cache.priceImpactAmount, cache.sizeDeltaInTokens, cache.executionPrice) = PositionUtils.getExecutionPriceForIncrease(params, prices.indexTokenPrice);
+        (cache.priceImpactUsd, cache.priceImpactAmount, cache.baseSizeDeltaInTokens, cache.executionPrice) = PositionUtils.getExecutionPriceForIncrease(params, prices.indexTokenPrice);
```

Comment:
maybe we should keep the naming as `sizeDeltaInTokens` instead of changing it to `baseSizeDeltaInTokens`